src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -957,64 +889,71 @@ subroutine Summary(this, bounds, &
     end associate
 
     ! total heterotrophic respiration (HR)
-       do fc = 1,num_soilc
-          c = filter_soilc(fc)
+    do fc = 1,num_soilc
+       c = filter_soilc(fc)
        
-          this%hr_col(c) = &
-               this%michr_col(c) + &
-               this%cwdhr_col(c) + &
-               this%lithr_col(c) + &
-               this%somhr_col(c)
+       this%hr_col(c) = &
+            this%michr_col(c) + &
+            this%cwdhr_col(c) + &
+            this%lithr_col(c) + &
+            this%somhr_col(c)
        
-       end do
+    end do
 
     ! Calculate ligninNratio
     ! FATES does its own calculation
-    if (.not. use_fates .and. decomp_method == mimics_decomp) then
-       do fp = 1,num_soilp
-          p = filter_soilp(fp)
-
-          associate(ivt => patch%itype)  ! Input: [integer (:)] patch plant type
-            ligninNratio_leaf_patch(p) = pftcon%lf_flig(ivt(p)) * &
-                                         pftcon%lflitcn(ivt(p)) * &
-                                         leafc_to_litter_patch(p)
-            ligninNratio_froot_patch(p) = pftcon%fr_flig(ivt(p)) * &
-                                          pftcon%frootcn(ivt(p)) * &
-                                          frootc_to_litter_patch(p)
-          end associate
-       end do
-
-       call p2c(bounds, num_soilc, filter_soilc, &
-            ligninNratio_leaf_patch(bounds%begp:bounds%endp), &
-            ligninNratio_leaf_col(bounds%begc:bounds%endc))
-       call p2c(bounds, num_soilc, filter_soilc, &
-            ligninNratio_froot_patch(bounds%begp:bounds%endp), &
-            ligninNratio_froot_col(bounds%begc:bounds%endc))
-       call p2c(bounds, num_soilc, filter_soilc, &
-            leafc_to_litter_patch(bounds%begp:bounds%endp), &
-            leafc_to_litter_col(bounds%begc:bounds%endc))
-       call p2c(bounds, num_soilc, filter_soilc, &
-            frootc_to_litter_patch(bounds%begp:bounds%endp), &
-            frootc_to_litter_col(bounds%begc:bounds%endc))
+    if_mimics: if (decomp_method == mimics_decomp ) then
```

Comment:
I don't mind how this is written, but what is the first `if_mimics:` statement for here, as opposed to 
`if (decomp_method == mimics_decomp ) then`?